app/waf/src/controllers/application.controller.ts

```diff
@@ -155,4 +173,79 @@ export class ApplicationController {
   async deleteById(@param.path.string('id') id: string): Promise<void> {
     await this.applicationRepository.deleteById(id);
   }
+
+  @post(prefix + '/applications/{id}/deploy', {
+    responses: {
+      '204': {
+        description: 'Application deploy success',
+      },
+    },
+  })
+  async deployById(@param.path.string('id') id: string): Promise<Object> {
+    let application = await this.applicationRepository.findById(id);
```

Comment:
Possible exception...

Reply:
Yes, but we needn't to catch it. The framework can return 404 error to client. It works in a same way as other handlers.

Reply:
ok